web/src/index.css

```diff
@@ -37,3 +37,28 @@ code {
   height: 50px;
   border-radius: 50%;
 }
+
+.product-name {
+  color: #1dc0ae;
+}
+
+.product-name:hover {
+  color: #1dc0ae;
+}
+
+.btn-primary, .btn-primary:hover, .btn-primary:active, .btn-primary:visited {
+    background-color: #1dc0ae !important;
+    border-color: #1dc0ae !important;
+}
+
+.btn-success, .btn-success:hover, .btn-success:active, .btn-success:visited {
+    background-color: #5463d2 !important;
+    border-color: #5463d2 !important;
+}
+
+.container a:last-child {
+  color: #5463d2 !important;
+}
+
+
+
```

Comment:
For colors we could do this: https://medium.com/swlh/creating-and-installing-a-custom-bootstrap-theme-with-react-2619e9aecfba

It looks like setting up SASS (.scss) shouldn't be too difficult. we'd just need to translate this file to SASS (or not) and use the short snippits from themestr.app to get this to work properly

Reply:
Added SASS @robotoer 
Here are the theme colors I ended up using. I made the "success" color the same as the "dark" background color just to avoid having an explosion of colors... 'info' is orange and 'light' is white, in case you want to use those ever.

<img width="306" alt="Screen Shot 2020-11-16 at 6 41 38 PM" src="https://user-images.githubusercontent.com/59711202/99339807-acaafa80-283b-11eb-91a8-2f51b5a5cf67.png">